src/dockerflow/logging.py

```diff
@@ -9,17 +9,31 @@
 import sys
 import traceback
 import uuid
+import warnings
 from contextvars import ContextVar
 from typing import ClassVar, Optional
 
 
+class MozlogHandler(logging.StreamHandler):
+    def __init__(self, stream=None, name="Dockerflow"):
+        if stream is None:
+            stream = sys.stdout
+        super().__init__(stream=stream)
+        self.logger_name = name
+        self.setFormatter(MozlogFormatter())
+
+    def emit(self, record):
+        record.logger_name = self.logger_name
+        super().emit(record)
```

Comment:
Could you maybe add a test for this part? ie. an assertion on the `"Logger"` field?